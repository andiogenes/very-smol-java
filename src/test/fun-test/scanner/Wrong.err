Error: unexpected "|" at 2:1
Error: unexpected "&" at 2:3
Error: unfinished exponential number literal "1." at 3:1
Error: unfinished exponential number literal "1.0" at 4:1
Error: unfinished exponential number literal "1.0E" at 5:1
Error:  "1.0E+" at 6:1
Error: unfinished exponential number literal "1.E" at 7:1
Error: unexpected "|" at 8:1
Error: unexpected "к" at 9:1
Error: unexpected "и" at 9:2
Error: unexpected "р" at 9:3
Error: unexpected "и" at 9:4
Error: unexpected "л" at 9:5
Error: unexpected "л" at 9:6
Error: unexpected "и" at 9:7
Error: unexpected "ц" at 9:8
Error: unexpected "а" at 9:9
Error: unexpected "№" at 10:1
Error: unexpected "#" at 11:1
Error: unclosed comment "/* unclosed comment" at 12:1