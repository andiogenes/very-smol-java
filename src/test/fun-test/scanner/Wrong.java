// Ошибочный тест (не менее 10 ошибок)
| &
1.
1.0
1.0E
1.0E+
1.E
|
кириллица
№
#
/* unclosed comment