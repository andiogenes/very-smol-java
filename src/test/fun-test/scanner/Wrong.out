Type: ERROR, lexeme: |, line: 2, pos: 1
Type: ERROR, lexeme: &, line: 2, pos: 3
Type: ERROR, lexeme: 1., line: 3, pos: 1
Type: ERROR, lexeme: 1.0, line: 4, pos: 1
Type: ERROR, lexeme: 1.0E, line: 5, pos: 1
Type: ERROR, lexeme: 1.0E+, line: 6, pos: 1
Type: ERROR, lexeme: 1.E, line: 7, pos: 1
Type: ERROR, lexeme: |, line: 8, pos: 1
Type: ERROR, lexeme: к, line: 9, pos: 1
Type: ERROR, lexeme: и, line: 9, pos: 2
Type: ERROR, lexeme: р, line: 9, pos: 3
Type: ERROR, lexeme: и, line: 9, pos: 4
Type: ERROR, lexeme: л, line: 9, pos: 5
Type: ERROR, lexeme: л, line: 9, pos: 6
Type: ERROR, lexeme: и, line: 9, pos: 7
Type: ERROR, lexeme: ц, line: 9, pos: 8
Type: ERROR, lexeme: а, line: 9, pos: 9
Type: ERROR, lexeme: №, line: 10, pos: 1
Type: ERROR, lexeme: #, line: 11, pos: 1
Type: ERROR, lexeme: /* unclosed comment, line: 12, pos: 1
Type: EOF, line: 12, pos: 19